src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,349 @@
+}
+
+#ic-demo video {
+margin: 0 0 20px 0;
+vertical-align: top;
+max-width: 100%;
+}
+
+</style>
+
+# Take Photos & Control Camera Settings {: .page-title }
+
+{% include "web/_shared/contributors/samdutton.html" %}
+
+Image Capture is an API to capture still images and configure camera hardware
+settings.
+
+The API enables control over camera features such as zoom, focus mode,
+contrast, ISO and white balance. Best of all, Image Capture enables access to
+the full camera resolution capabilities. Previous techniques
+for taking photos on the Web have used snapshots from a video stream, which are
+lower resolution than those available for still images.
+
+Image Capture is available in Chrome 56 on desktop and Android as an
+[Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md),
+or in Chrome Canary on desktop and Android with **Experimental Web Platform**
+features enabled.
+
+The API has four methods:
+
+* `takePhoto()` returns a `Blob`, [the result of a single photographic exposure](https://www.w3.org/TR/image-capture/#dom-imagecapture-takephoto),
+which can be downloaded, stored by the browser or displayed in an `img` element.
+This method uses the highest available photographic camera resolution.
+* `grabFrame()` returns an `ImageBitmap` object,
+[a snapshot of live video](https://www.w3.org/TR/image-capture/#dom-imagecapture-grabframe),
+which could (for example) be drawn on a canvas and then post-processed to
+selectively change color values. Note that the `ImageBitmap` will only have the
+resolution of the video source — which will be lower than the camera's
+still-image capabilities.
+* `getPhotoCapabilities()` returns a `PhotoCapabilities` object that provides
+access to available camera options and their current values.
+* `setOptions()` is used to configure
+[camera settings](http://www.w3.org/TR/image-capture/#photosettings) such as
+zoom, white balance or focus mode.
+
+The Image Capture API gets access to a camera via a `MediaStream` from `getUserMedia()`:
+
+    navigator.mediaDevices.getUserMedia({video: true})
+      .then(gotMedia)
+      .catch(error => console.error('getUserMedia() error:', error));
+
+    function gotMedia(mediaStream) {
+      const mediaStreamTrack = mediaStream.getVideoTracks()[0];
+      const imageCapture = new ImageCapture(mediaStreamTrack);
+      console.log(imageCapture);
+    }
+
+You can try out this code from the DevTools console.
+
+Note: To choose between different cameras, such as the front and back camera on
+a phone, get a list of available devices via the
+`MediaDevices.enumerateDevices()` method, then set `deviceId` in `getUserMedia()`
+constraints — as per the demo [here](https://webrtc.github.io/samples/src/content/devices/input-output/).
+
+You can use `takePhoto()` to get a still image and then set it as the `src` of
+an `<img>`:
+
+    imageCapture.takePhoto()
+      .then(blob => {
+        image.src = URL.createObjectURL(blob);
```

Comment:
So is `image` the `<img>` tag? Retrieved perhaps with `querySelector()` or some such thing? Call it a nit, but I like that level of clarity.